content/docs/platform/additional-resources/errors.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Errors'
+description: 'Common errors and their solutions'
+icon: 'alert-circle'
+---
+
+## Introduction
+
+Activity feed page displays all the events that have occurred This page is useful for debugging and monitoring triggered events and notification delivery. Each event has steps and logs with each step. These logs can have errors which can be useful for debugging.
+
+## Common errors
+
+### Webhook URL for the chat channel is missing
```

Comment:
Are those names are exactly like printed on the activity feed? Just if someone will copy and search for them